ament_cpplint/ament_cpplint/main.py

```diff
@@ -144,7 +148,8 @@ def main(argv=sys.argv[1:]):
         else:
             print("Not using '--root'")
         print('')
-        arguments += files
+        arguments += [file for file in files
```

Comment:
Nitpick: `file` is a builtin and shouldn't be used as a variable name (I would guess this also results in a linter warning).